Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -354,6 +354,8 @@ int main(int argc, char** argv)
     };
 
     process_table_view.on_selection_change = [&] {
+        if (!process_table_view.is_focused())
+            return;
```

Comment:
It is possible to lose focus from the process table view without actually switching away (click on the process table view and then press shift+tab, for instance). Will it no longer update in this case?